apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/server/S3JsonMessageSources.java

```diff
@@ -20,6 +20,7 @@
 
   private final AmazonS3 s3Client;
   @Getter private final String bucketName;
+  @Getter private final String directoryPath;
```

Comment:
Does this not require an update to where these are created?

Reply:
Previously this wound up being used to build the `fissPrefix` and `mcsPrefix` values so it wasn't stored in a field.  Now I'm storing it in a field and using the prefix passed by the caller without modification.  So the caller of the constructore didn't have to have any changes.

Reply:
Ah yes, sorry, misread the change